packages/hooks/src/__tests__/useAuthenticationStatusInfo.spec.tsx

```diff
@@ -0,0 +1,304 @@
+import * as React from 'react';
+import { renderHook } from '@testing-library/react-hooks';
+import { APIProvider } from '@deriv/api';
+import useAuthenticationStatusInfo from '../useAuthenticationStatusInfo';
+import useAccountStatus from '../useAccountStatus';
+
+jest.mock('../useAccountStatus.ts', () => jest.fn());
+
+const mockUseAccountStatus = useAccountStatus as jest.MockedFunction<typeof useAccountStatus>;
```

Comment:
Don't mock internals, Just mocking `useFetch` for `get_account_status` is enough for writing tests.